lib/fluent/plugin/out_color_stripper.rb

```diff
@@ -8,8 +8,6 @@ class ColorStripperOutput < Output
     def configure(conf)
       super
 
-      @tag = conf.fetch('tag') { raise ArgumentError, 'tag field is required to direct transformed logs to' }
```

Comment:
`config_param` without `default` option will be required parameter.
Fluentd checks tag is present or not on boot.